pkg/storage/prunable/prunable_slot.go

```diff
@@ -52,6 +53,11 @@ func (p *Prunable) RootBlocks(slot iotago.SlotIndex) (*slotstore.Store[iotago.Bl
 	), nil
 }
 
+func (p *Prunable) CommitteeCandidates(epoch iotago.EpochIndex) (kvstore.KVStore, error) {
+	// Slot 0 does not exist, but we need to add it to the prefix to avoid random clashes with other keys.
+	return p.prunableSlotStore.Get(epoch, byteutils.ConcatBytes(iotago.SlotIndex(0).MustBytes(), kvstore.Realm{epochPrefixCommitteeCandidates}))
```

Comment:
this is hacky, better use the epoch startSlot so it at least matches the other stores.